deployment/terraform/assets/cluster.tf

```diff
@@ -424,6 +465,12 @@ resource "aws_security_group" "app" {
     protocol        = "tcp"
     security_groups = [aws_security_group.metrics[0].id]
   }
+  ingress {
+    from_port   = 2049
+    to_port     = 2049
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"] # Allow NFS traffic within VPC
```

Comment:
Can we restrict this some more? It wouldn't be the first time customers complain about permissions being too open when they run the tool.